lonestar/mining/cpu/frequent-subgraph-mining/README.md

```diff
@@ -16,12 +16,12 @@ of the Meeting on Algorithm Engineering & Expermiments, 135-149.
 INPUT
 --------------------------------------------------------------------------------
 
-We support the following four input graph formats: **gr**, **txt**, **adj**, **mtx**.
+We support the following input graph formats: **txt**, **adj**.
 
-We mostly use **adj** format as also used by Arabesque and RStream.
+We mostly use **adj** format as it is also used by Arabesque and RStream.
 The **adj** format takes as input graphs with the following formats:
 
-* **Graphs label on vertex(default)**
+* **Graph labels on vertices (default)**
```

Comment:
I do not understand. Do you mean "graph with labels on vertices" or "vertex labels"?

Reply:
labels on vertices

Reply:
Then change it to "Graph with labels on vertices"

Reply:
fixed